sdjournal/journal.go

```diff
@@ -835,6 +837,7 @@ func (j *Journal) GetMonotonicUsec() (uint64, error) {
 // GetCursor gets the cursor of the current journal entry.
 func (j *Journal) GetCursor() (string, error) {
 	var d *C.char
+	defer C.free(unsafe.Pointer(d))
```

Comment:
This (together with the var declaration perhaps?) should be moved past the `getFunction` call, or it may be freeing a pointer that has never gone through a malloc.